compiler/build.gradle

```diff
@@ -75,6 +75,7 @@ model {
   binaries {
     all {
       if (toolChain in Gcc || toolChain in Clang) {
+        cppCompiler.define("GRPC_VERSION", version)
```

Comment:
(According to docs, it should work - https://docs.gradle.org/current/userguide/native_software.html, see `72.12. Configuring the compiler, assembler and linker` )